### PR TITLE
chore: remove development configuration before publishing package

### PR DESCRIPTION
**Pull request by ychhabra-eightfold**
## SUMMARY:
chore: remove development configuration before publishing package

## CHANGE TYPE:

-   [ ] Bugfix Pull Request
-   [ ] Feature Pull Request

## TEST COVERAGE:

-   [ ] Tests for this change already exist
-   [ ] I have added unittests for this change

### CODE REVIEW

File: clean-package.config.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "remove": ["engines", "lint-staged", "jest", "eslintConfig", "scripts"]
```
Comment (darora-eightfold) [inline clean-package.config.json:2]: We want to undo fields other than engines?

**Review by darora-eightfold — APPROVED** [review]
LGTM

**Comment by darora-eightfold** [comment]
needs_sandbox

File: clean-package.config.json (hunk 1 above)
Comment (ychhabra-eightfold) [reply clean-package.config.json:2]: All fields are development config dont make sense to be in the published package

**Review by jhoward-eightfold — APPROVED** [review]

**Pull request merged**